package.json

```diff
@@ -124,7 +132,7 @@
 			".+\\.(css|styl|less|sass|scss|png|jpg|ttf|woff|woff2)$": "jest-transform-stub"
 		},
 		"transformIgnorePatterns": [
-			"/node_modules/(?!vue-material-design-icons)"
+			"/node_modules/(?!(.*)/)"
```

Comment:
Does someone know how to configure this to transform the dependencies brought in by `NcRichText` without disabling it completely or having to list all sub-dependencies manually?

Reply:
Maybe @juliushaertl @skjnldsv?

Reply:
I think listing them is the most reasonable

Reply:
> I think listing them is the most reasonable

I tried that, but after arriving at a still incomplete list of over 10 packages, I figured it's unreasonable and aborted. 

I guess we can come up with a complete list that works, but I am afraid that the list might change even with only a minor update of one of the packages.

Reply:
There shouldn't be a downside of just transforming all, except performance of test runs, right?

Reply:
Yes, it's only about the performance. The tests run anyway.
But it does seem to have quite an impact. Before the tests more or less took 7s. This test run here now took 24s, of which 5s are for the added RichText tests, so its 7s vs 19s. Nearly three times as much. It could be an outlier though, I just checked one test run and also not locally though.

Reply:
@raimund-schluessler Any objection to merge this?

Reply:
@julien-nc Well, it's not my GH actions minutes, so feel free to go ahead 🙈😄 We can of course still optimize this later, once we find a suitable solution.